src/makefile.all

```diff
@@ -1495,43 +1495,43 @@ DEBUGRC = 0
 
 @ifdef BORLAND
 $(RESOURCE): watt-32.rc
-        $(BRC32) -DDEBUG=0 -D__BORLANDC__ -r -i..\inc -fo $(RESOURCE) watt-32.rc
+	$(BRC32) -DDEBUG=0 -D__BORLANDC__ -r -i..\inc -fo $(RESOURCE) watt-32.rc
```

Comment:
CBuilder or Borland's Make do accept `<tab>` in rules. 

If that was used for all targets, it would be better. Are there `<tab>` in all rules in this PR?

Reply:
watcom has no proble with <tab>, on some places it is required.
If all others accept is then it is OK to use it for all.